app/controllers/categories_controller.rb

```diff
@@ -1,19 +1,25 @@
 class CategoriesController < ApplicationController
   before_action :authenticate_user!
-  before_action :find_category, only: %i[update destroy edit category_history]
+  before_action :find_category, only: %i[update destroy edit show]
+  before_action :set_parent, only: %i[new create]
 
   def index
-    @categories = current_user.categories.paginate(page: params[:page], per_page: 10)
+    @categories = current_user.categories.top_category.paginate(page: params[:page], per_page: 10)
+  end
+
+  def show
+    @transactions = current_user.transactions.category_history_transactions(params[:id].to_i)
```

Comment:
@sinpai Q about `to_i` from previous PRs

Reply:
http://take.ms/nGVyy - it's a string. But I need integer in this case.